app/src/main/java/com/example/admin/nyproject/ui/main/view/MainFragment.java

```diff
@@ -105,30 +135,86 @@ protected void initView() {
     void onCreateButtonClickListener() {
         if (mNavigator != null) {
             mNavigator.showSaveFragment();
+            Bundle bundle = new Bundle();
+            bundle.putString("cubes", cubesOfpacket );
+            bundle.putString("quantity", quantityOfBoars);// set your parameteres
+
+            SaveFragment nextFragment = new SaveFragment();
+            nextFragment.setArguments(bundle);
+
+            FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+            fragmentManager.beginTransaction().replace(R.id.framelayout_activity_main_jaf_container, nextFragment).commit();
+
+
         }
     }
 
     @OnClick(R.id.btnRestore)
     void onRestoreButtonClickListener() {
+        restoreData();
```

Comment:
Could be 
`void onRestoreClickListener(){
mPresenter.restoreData();
}`

No need to create `resotreData` method